import_tables.sql

```diff
@@ -22,15 +22,16 @@ SELECT
 FROM source_pluto
 INNER JOIN borough ON source_pluto.borough=borough.abbr;
 
-COPY zoning_district (
-	"id",
-	"label",
-	"wgs84",
-	"li_ft"
-)
-	FROM '../zoning_district.csv'
-	DELIMITER ','
-	CSV HEADER;
+INSERT INTO zoning_district
+SELECT
+    GEN_RANDOM_UUID() AS id,
+    zonedist AS label,
+    ST_Transform(wkt, 4326) as wgs84,
```

Comment:
It seems like the `wkt` column in source_zoning_districts is already in wgs84 so I don't think we need to transform it again. (This is the opposite of how populated the tax_lot table)

Reply:
Good call